packages/components/app/styles/components/table.scss

```diff
@@ -2,107 +2,173 @@
 // TABLE
 //
 //
+$hds-table-border-radius: 6px;
+$hds-table-border-width: 1px;
+$hds-table-inner-border-radius: $hds-table-border-radius - $hds-table-border-width;
+$hds-table-border-color: var(--token-color-border-primary);
+
 .hds-table,
 .hds-table-sortable {
   width: 100%;
   font-size: var(--token-typography-body-200-font-size);
   font-family: var(--token-typography-font-stack-text);
   line-height: var(--token-typography-body-200-line-height);
   table-layout: fixed;
-  border: 1px solid var(--token-color-border-primary);
-  border-radius: 6px;
-  border-collapse: collapse;
+  border: $hds-table-border-width solid $hds-table-border-color;
+  border-radius: $hds-table-border-radius;
   border-spacing: 0;
 }
 
+// TABLE HEADER
+.hds-table__thead {
+  .hds-table__tr {
+    color: var(--token-color-foreground-strong);
+    background-color: var(--token-color-surface-strong);
+
+    .hds-table__th,
+    .hds-table__th-sort {
+      height: 48px;
+      color: var(--token-color-foreground-strong);
+      font-weight: var(--token-typography-font-weight-semibold);
+      font-size: 14px;
+      line-height: 1.4;
+      text-align: left;
+      background-color: var(--token-color-surface-strong);
+      border-top: none;
+      border-right: $hds-table-border-width solid $hds-table-border-color;
+      border-bottom: $hds-table-border-width solid $hds-table-border-color;
+      border-left: none;
```

Comment:
I experimented with simplified styles fro borders and border radius. The main drawback is that it requires an extra wrapper div around the table: https://codepen.io/KristinB/pen/rNKxxPz/ec9d9ad82552b4794d7cea417b71936e
@MelSumner

Reply:
Thanks for sharing! I considered that approach, but overall I tend to lean toward less markup, since there is a limit to how many DOM nodes can exist before a screen reader will crash (and this is pretty easy to do in technical applications).